### PR TITLE
[Tests][Component] Add tests to src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx

### LINKED ISSUE #219: [Tests][Component] Add tests to src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx
### Context
once a user successfully creates an bounty it's details are provide by `CodingBounty.tsx`

### File to to test
`src/people/widgetViews/summaries/wantedSummaries/CodingBounty`

### File to create
`src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx`

### Acceptance Criteria
- [ ] Test that on clicking on "not assigned", a pop up should appear to invite a developer including "type to search" box, a"skills" box, and a recommendation of 5 hunters.
- [ ] Test that on clicking on the "type to search" box I should be able to type in any character and the list below should be filtered.
- [ ] Test that on clicking on the "skills" box I should be able to type in any character and the list below should be filtered.
- [ ] Test that on clicking on "invite" on a hunter, the pop up should clear and the hunter should be assigned to the bounty.
- [ ] I have rebased and tested locally before submitting my PR

### References
- Watch this [Jest Youtube playlist](https://www.youtube.com/watch?v=T2sv8jXoP4s&list=PLC3y8-rFHvwirqe1KHFCHJ0RqNuN61SJd) if you are new to testing
- Here is an [example unit test](https://github.com/stakwork/sphinx-tribes/blob/master/frontend/app/src/helpers/__test__/helpers.spec.ts)
- Here is an [example component test](https://github.com/stakwork/sphinx-tribes/blob/9310f49b3b17a51992dada932f4298eb9eba15ff/frontend/app/src/people/widgetViews/__tests__/AboutView.spec.tsx)

### PR DESCRIPTION
**Pull request by arimusdev**
## Describe your changes

Add tests to `src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx`
image

## Issue ticket number and link

closes #219 

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by aliraza556** [comment]
Hi @arimusdev,
Well done! The screenshot looks great. Now, let's take a moment to review the code.

**Comment by aliraza556** [comment]
Hi @arimusdev,
Impressive work on the test suite! It's thorough, well-structured, and showcases strong testing skills. Great job ensuring comprehensive coverage and maintaining code quality. Keep it up!

**Comment by arimusdev** [comment]
> Hi @arimusdev, Impressive work on the test suite! It's thorough, well-structured, and showcases strong testing skills. Great job ensuring comprehensive coverage and maintaining code quality. Keep it up!

Thanks, Ali 😀

File: src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx [hunk 1]
```diff
@@ -163,4 +171,158 @@ describe('MobileView component', () => {
...
+
+    expect(document.querySelectorAll('.PeopleList .People')).toHaveLength(1);
+    expect(screen.getByText('TEST_NAME_1')).toBeInTheDocument();
+    expect(screen.queryByText('TEST_NAME_2')).not.toBeInTheDocument();
+    expect(screen.queryByText('TEST_NAME_3')).not.toBeInTheDocument();
+
+    mockSearch.mockRestore();
+  });
+
+  it('Test that on clicking on "invite" on a hunter, the pop up should clear and the hunter should be assigned to the bounty', async () => {
```
Comment (elraphty) [inline src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx:279]: @arimusdev Can you change the description of this test from "Invite" to  "Assign"

**Comment by ecurrencyhodler** [comment]
Paid 200k. GJ! https://community.sphinx.chat/bounty/1466

**Pull request merged**